### PR TITLE
Added download-header component

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: src/download-header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  switch (licenceType) {
+    case 'ppl':
+      title = model.title || 'Untitled project';
+      break;
+    case 'pil':
+      title = 'Personal licence';
+      break;
+    case 'pel':
+      title = 'Establishment licence';
```
Comment (lennym) [inline src/download-header/index.jsx:17]: ~Is this in the designs? It feels weirdly inconsistent to use the project title for projects, but not the establishment name for establishments?~

Yes.

**Comment by lennym** [comment]
While we're here, is it possible to fix one my major irritations and have clicking a download link also close the menu?

Note: if you do this then you'll need to remove this line: https://github.com/UKHomeOffice/asl-deployments/blob/master/tests/public/helpers/project.js#L164

File: src/download-header/index.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      right: 1em;
+      text-decoration: none;
+    }
+  }
+
+  .download {
+    font-size: 1.2em;
+    margin-left: 0.5em;
+    &:before {
+      @include icon;
```
Comment (joefitter) [inline src/download-header/index.scss:48]: is this a thing outside of projects? If not then we should move it a level up so components doesn't implicitly depend on projects

Comment (joefitter) [reply src/download-header/index.scss:48]: ah yeah, I just seen it below - however it will be defined in 2 places now I'm guessing?

Comment (wheelsandcogs) [reply src/download-header/index.scss:48]: I've added the mixin here: https://github.com/UKHomeOffice/asl-components/pull/85/files#diff-e30d20921c2abad49a8461ea88b2c052R11

File: src/download-header/index.jsx (hunk 1 above)
Comment (wheelsandcogs) [reply src/download-header/index.jsx:17]: Yup.

File: src/download-header/index.scss (hunk 2 above)
Comment (joefitter) [reply src/download-header/index.scss:48]: fine fine fine, nobody cares about css anyway

Comment (lennym) [reply src/download-header/index.scss:48]: Yes. https://github.com/UKHomeOffice/asl-components/pull/85/files#diff-e30d20921c2abad49a8461ea88b2c052R11-R19

**Comment by wheelsandcogs** [comment]
> While we're here, is it possible to fix one my major irritations and have clicking a download link also close the menu?
> 
> Note: if you do this then you'll need to remove this line: https://github.com/UKHomeOffice/asl-deployments/blob/master/tests/public/helpers/project.js#L164

Done.

The test should be fine as-is because we don't actually click the download links, we just open the modal to grab the href.

**Comment by lennym** [comment]
Yeah, it was mostly just as a note that if the tests start failing then that's likely why. Any test that downloads twice without moving away will have issues, but I don't think such a thing exists.

**Pull request merged**